.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: goreleaser
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-20.04
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      -
+        name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.15
+      -
+        name: Run GoReleaser
+        uses: goreleaser/goreleaser-action@v2
+        with:
+          version: latest
+          args: release --rm-dist
```

Comment:
If we proceed with incorporating goreleaser in the repository  there's one problem with breaking backward compatibility (to some degree). The assets' name and link will change:

```
current      : https://github.com/git-chglog/git-chglog/releases/download/0.9.1/git-chglog_linux_amd64
new approach : https://github.com/git-chglog/git-chglog/releases/download/0.9.2/git-chglog_0.9.2_Linux_x86_64.tar.gz
```
Just to be clear, I'm not rejecting the idea of adding goreleaser, but rather either 1) try to keep the naming same as before or 2) clearly broadcast the this new change.

Reply:
Good point. We could easily adapt this with an `name_template` from the [archive](https://goreleaser.com/customization/archive/) section, but also regarding the https://github.com/git-chglog/git-chglog/issues/88#issuecomment-760996234 I favour switching to a schema like `v0.10.0` and communicate it clearly. Fine for you?

Reply:
I remember having a bit of difficulty in the past making `name_template` of goreleaser work exactly like I wanted, specifically keeping the naming of uncompressed file. Note that goreleaser, of out of the box, will create final assests of compressed (`tar.gz` and `zip`) file. All thing considered I prefer, if possible, publishing the actual binaries and not the compressed files.

But I think that's the best approach for us anyway. I think trying to keep it as close as the current state is ideal. Something like:

```
https://github.com/git-chglog/git-chglog/releases/download/v<VERSION>/git-chglog_<OS>_<ARCH>
```

where:

- `VERSION`: is the tag with leading `v`
- `OS`: lowercased version of operating system
- `ARCH`: the equivalent of `GOARCH` (i.e. `amd64` instead of `x86_64`)

Just took another look, and effectively removing `archive.replacements` section of `.goreleaser.yml`.

Reply:
That being said, I'm completely fine with breaking a bit of naming if this is proven to be more difficult to implement than expected. Completely your call.

Reply:
>  I think trying to keep it as close as the current state is ideal.

I'm in on that and removes the `replacements` section for archives. 

> All thing considered I prefer, if possible, publishing the actual binaries and not the compressed files.

When I checked the resulting `git-chglog.rb` brew formula, it refers to the archives. So I kept them in.

Reply:
Sounds good!